hd/hd_test.go

```diff
@@ -237,6 +146,6 @@ func BenchmarkReverseBytes(b *testing.B) {
 
 	// sink is necessary to ensure if the compiler tries
 	// to smart, that it won't optimize away the benchmarks.
-	if sink != nil {
+	if sink != nil { // nolint: megacheck
```

Comment:
better to use code than comments. here we can just put `_ = sink` in the if block